CMakeLists.txt

```diff
@@ -1,63 +1,86 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
the version on ubunut18.04 is 3.10 btw, it does change anything really but just for you to know.
This is why I have put 3.10 everywhere.